src/Mvc/global.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
@ryanbrandenburg @natemcmaster should global.json files that are not in the aspnet/AspNetCore root directory be removed in general?